Assets/Puppet/PuppetAppCenter.cs

```diff
@@ -287,6 +303,32 @@ public void SetAppSecret(string appSecret)
 #endif
     }
 
+    public void SetUpdateTrack()
+    {
+        int updateTrack = UpdateTrackDropDown.value + 1;
```

Comment:
`+ 1`?
If that works it looks like a hack, we should have more readable code to switch the value.

Reply:
Fixed